### ISSUE TITLE
update config for nginx

### ISSUE THREAD

**Issue by omansour** [M6Web/PhpProcessManagerBundle#12]
as @bdu-p recomands : 

```
[09:39] BD: perso pour de la prod j'ajouterai quelques directive pour le nginx
[09:40] BD: proxy_connect_timeout  20;
        proxy_send_timeout        20;
        proxy_read_timeout        20;
        proxy_next_upstream       http_500 http_502 http_503 http_504 timeout error;
        et pour chaque upstream
        ajouter
[09:40] BD: max_fails=2 fail_timeout=20;
[09:40] BD:    server 127.0.0.1:8000 max_fails=2 fail_timeout=20;
```

**Comment by fabdsp** [comment]
https://github.com/M6Web/PhpProcessManagerBundle/pull/14

**Comment by mikaelrandy** [comment]
closed as #14 is closed too.
